aws-xray-propagator/gradle.properties

```diff
@@ -0,0 +1,2 @@
+# TODO: uncomment when ready to mark as stable
+# otel.stable=true
```

Comment:
As it stands, `io.opentelemetry:opentelemetry-extension-aws` is downgraded from stable to alpha.